File: Modul-3/C/PRAK302-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
  int nilai;
  char grade;

  printf("Masukkan nilai: ");
  scanf("%i", &nilai);

  if (nilai >= 80)
  {
    grade = 'A';
  }
  else if (nilai >= 70 && nilai <= 79)
  {
    grade = 'B';
  }
  else if (nilai >= 60 && nilai <= 69)
  {
    grade = 'C';
  }
  else if (nilai >= 50 && nilai <= 59)
  {
    grade = 'D';
  }
  else if (nilai < 50)
  {
    grade = 'E';
  }

  printf("Grade anda: %c\n", grade);

  return 0;
}

File: Modul-3/Python/PRAK302-2310817110006-Adrian_Bintang_Saputera.py
nilai = int(input("Masukkan nilai: "))

if nilai >= 80:
    grade = 'A'
elif nilai >= 70 and nilai <= 79:
    grade = 'B'
elif nilai >= 60 and nilai <= 69:
    grade = 'C'
elif nilai >= 50 and nilai <= 59:
    grade = 'D'
else:
    grade = 'E'

print(f"Grade anda: {grade}")